Bottom: 1f10aabdf1fb229a626c0726bb852a0c04fe690f
Top:    5fc402168b3b0d1c45a2fa7499db7f83677f8e13
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-07-12 14:07:18 -0500

Refresh of generate-new-ssh-key-on-first

---
diff --git a/roles/buildbot-master/tasks/main.yml b/roles/buildbot-master/tasks/main.yml
index 8a2ac0be..d7d69e36 100644
--- a/roles/buildbot-master/tasks/main.yml
+++ b/roles/buildbot-master/tasks/main.yml
@@ -36,7 +36,7 @@
 
 - name: Grab ssh public key
   command: >
-    docker exec '{{ buildbot_container_name }}' cat /home/buildbot/.ssh/id_rsa.pub
+    docker exec '{{ container_name }}' cat /home/buildbot/.ssh/id_rsa.pub
   changed_when: false
   register: output
